modules/xmpp/RtxModifier.js

```diff
@@ -120,17 +120,30 @@ export default class RtxModifier {
 
             return sdpStr;
         }
+
+        return this.modifyRtxSsrcs2(videoMLine)
+            ? sdpTransformer.toRawSDP() : sdpStr;
+    }
+
+    /**
+     * Does the same thing as {@link modifyRtxSsrcs}, but takes the
+     *  {@link MLineWrap} instance wrapping video media as an argument.
+     * @param {MLineWrap} videoMLine
+     * @return {boolean} <tt>true</tt> if the SDP wrapped by
+     *  {@link SdpTransformWrap} has been modified or <tt>false</tt> otherwise.
+     */
+    modifyRtxSsrcs2(videoMLine) {
```

Comment:
but by reading `modifyRtxSsrcs2` it's not clear how that method differs from `modifyRtxSsrcs`, which i think is weird.  js doesn't support method overloading, right?  it'd actually be clearer if they could have the same name i think.  maybe this should be 'modifyVideoMLineRtxSsrcs` or something.  or, this could be called `modifyRtxSsrcs` and the original one could be renamed.